### PR TITLE
Added merchant screening

### PR DESCRIPTION
**Pull request by Omri-Levy**
- feat(backoffice-v2): replaced pagination component and added last page button
- revert(backoffice-v2): removed unused isEmptyArray util
- fix(backoffice-v2): removed sleep added by mistake
- feat(*): moved merchant screening processing to the unified api
- feat(*): updated packages

## Summary by CodeRabbit

- **Bug Fixes**
	- Updated dependencies across multiple packages to ensure compatibility and stability.

- **Documentation**
	- Updated changelogs for various packages to reflect new version entries and dependency changes.

- **Chores**
	- Incremented version numbers for multiple packages to indicate new releases.

### CODE REVIEW

File: packages/ui/src/components/atoms/Image/Image.tsx [hunk 1]
```diff
@@ -3,16 +3,21 @@ import { BaseImage } from '@/components/atoms/Image/BaseImage';
...
           <figure
-            className={`border-destructive flex flex-col items-center justify-center space-y-2 rounded-md border p-1`}
-            style={{ width, height }}
             aria-live={`polite`}
+            {...props}
+            className={ctw(
+              `border-destructive flex flex-col items-center justify-center space-y-2 rounded-md border p-1`,
+              props?.className,
+            )}
+            style={{ width, height, ...props?.style }}
```
Comment (tomer-shvadron) [inline packages/ui/src/components/atoms/Image/Image.tsx:20]: Wouldn't it be better to extract `style` from props, the same way we do for `width` and `height`?

File: packages/workflow-core/src/lib/constants.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+} as const;
+
+export const pluginsRegistry = {
+  [PluginKind.KYC]: KycPlugin,
+  [PluginKind.KYC_SESSION]: KycSessionPlugin,
+  [PluginKind.KYB]: KybPlugin,
+  [PluginKind.WEBHOOK]: WebhookPlugin,
+  [PluginKind.API]: ApiPlugin,
+  [PluginKind.EMAIL]: EmailPlugin,
+  [PluginKind.MASTER_CARD_MERCHANT_SCREENING]: MasterCardMerchantScreeningPlugin,
```
Comment (tomer-shvadron) [inline packages/workflow-core/src/lib/constants.ts]:
```suggestion
  [PluginKind.MASTERCARD_MERCHANT_SCREENING]: MastercardMerchantScreeningPlugin,
```

File: packages/workflow-core/src/lib/constants.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { ObjectValues } from './types';
+
+export const PluginKind = {
+  KYC: 'kyc',
+  KYC_SESSION: 'kyc-session',
+  KYB: 'kyb',
+  WEBHOOK: 'webhook',
+  API: 'api',
+  EMAIL: 'email',
+  MASTER_CARD_MERCHANT_SCREENING: 'mastercard-merchant-screening',
```
Comment (tomer-shvadron) [inline packages/workflow-core/src/lib/constants.ts]:
```suggestion
  MASTERCARD_MERCHANT_SCREENING: 'mastercard-merchant-screening',
```

File: packages/workflow-core/src/lib/plugins/external-plugin/master-card-merchant-screening-plugin.ts [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        principals: [
+          {
+            firstName: entity?.data?.additionalInfo?.mainRepresentative?.firstName,
+            lastName: entity?.data?.additionalInfo?.mainRepresentative?.lastName,
+            address,
+          },
+        ],
+      };
+
+      logger.log('API Plugin - Sending API request', {
```
Comment (tomer-shvadron) [inline packages/workflow-core/src/lib/plugins/external-plugin/master-card-merchant-screening-plugin.ts]: Update log to `Mastercard Plugin` in this file

File: packages/workflow-core/src/lib/plugins/external-plugin/master-card-merchant-screening-plugin.ts [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+          'Request Failed: ' + apiResponse.statusText + ' Error: ' + JSON.stringify(errorResponse),
+        );
+      }
+    } catch (error) {
+      logger.error('Error occurred while sending an API request', { error });
+
+      return this.returnErrorResponse(isErrorWithMessage(error) ? error.message : '');
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (tomer-shvadron) [reply packages/workflow-core/src/lib/plugins/external-plugin/master-card-merchant-screening-plugin.ts]: 👍

File: packages/workflow-core/src/lib/plugins/external-plugin/webhook-plugin.ts [hunk 6]
```diff
@@ -18,7 +19,57 @@ export class WebhookPlugin extends ApiPlugin {
     }
 
     try {
-      await this.makeApiRequest(this.url, this.method, requestPayload, this.headers!);
+      const urlWithoutPlaceholders = await this.replaceValuePlaceholders(this.url, context);
+
+      logger.log('API Plugin - Sending API request', {
```
Comment (tomer-shvadron) [inline packages/workflow-core/src/lib/plugins/external-plugin/webhook-plugin.ts]: Update log to Webhook Plugin in this file

**Review by tomer-shvadron — APPROVED** [review]

File: packages/ui/src/components/atoms/Image/Image.tsx (hunk 1 above)
Comment (Omri-Levy) [reply packages/ui/src/components/atoms/Image/Image.tsx:20]: Difference is `width` and `height` are props I added myself to make them required as opposed to the original optional. Up to you if you still think there's value in doing so.

**Pull request merged**